### PR TITLE
Improve visual styling & clarity

### PR DESCRIPTION
**Pull request by callumflack**
* add a step marker to the Prompt
* use a stronger display serif
* reuse Spinner for awaiting generated images placeholder
* add info to prompt CTA
* improve layout & CSS spacing

### CODE REVIEW

**Comment by callumflack** [comment]
Looks like:
image

**Comment by callumflack** [comment]
@zahay117 I don't want to go to far into design on this repo because a) we're back to designing in code, which limits our range of possibilities, b) we don't have much time and c) we don't want the codebase to get too big and hard to read.

The above UI is understandable and nice to look at.

File: components/prompt/Marker.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <path
+              fill-rule="evenodd"
+              clip-rule="evenodd"
+              d="M11.7736 17L6.00006 27L0.226553 17L5.00006 17L5.00006 -2.96079e-07L7.00006 -2.08657e-07L7.00006 17L11.7736 17Z"
+              fill="currentColor"
+            />
+          </svg>
+        </span>
+      )} */}
+
```
Comment (dmitryvlass) [inline components/prompt/Marker.js:24]:
```suggestion
```

File: pages/_app.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import "styles/globals.css";
-import { Inter, Spectral } from "@next/font/google";
+import { Inter, Source_Serif_Pro } from "@next/font/google";
 
 const inter = Inter({ subsets: ["latin"] });
-const spectral = Spectral({ subsets: ["latin"], weight: '600' });
+const source = Source_Serif_Pro({ subsets: ["latin"], weight: '900' });
```
Comment (dmitryvlass) [inline pages/_app.js:5]: 👍

**Review by dmitryvlass — APPROVED** [review]

**Pull request merged**